zgw/documenten-api/src/main/kotlin/com/ritense/documentenapi/service/DocumentenApiService.kt

```diff
@@ -61,6 +64,30 @@ class DocumentenApiService(
         documentApiPlugin.deleteInformatieObject(documentUrl)
     }
 
+    fun getColumns(caseDefinitionName: String): List<DocumentenApiColumn> {
+        return documentenApiColumnRepository.findAllByIdCaseDefinitionNameOrderByOrder(caseDefinitionName)
+    }
+
+    fun updateColumnOrder(columns: List<DocumentenApiColumn>): List<DocumentenApiColumn> {
+        val existingColumns = documentenApiColumnRepository.findAllByIdCaseDefinitionNameOrderByOrder(columns[0].id.caseDefinitionName)
```

Comment:
calling `columns[0]` without checking if it's not empty seems risky.

Reply:
I've added a check